config/locales/en.yml

```diff
@@ -38,7 +38,7 @@ en:
         public_domain_text: 'Access digital copy through HathiTrust'
         restricted_access_text: 'Search inside at HathiTrust'
         etas_text: 'Check out digital copy through HathiTrust'
-        etas_additional_text: 'By special arrangement in response to the COVID-19 pandemic, patrons may access this resource online through the HathiTrust Emergency Temporary Access Service.'
+        etas_additional_text: 'Digital access to this item through the HathiTrust Emergency Temporary Access Service will conclude on July 9, 2021. Requests for most library materials will be possible through "I Want It." On-shelf access will resume as libraries fully re-open over the summer. We will retain digital access to public domain materials in HathiTrust.'
```

Comment:
A nice yaml trick is to use ">" which allows line breaks. It really makes it easier to read the diffs:

``` yaml
etas_additional_text: >
  Digital access to this item through the HathiTrust Emergency Temporary Access Service will conclude on
  July 9, 2021. Requests for most library materials will be possible through "I Want It." On-shelf access will
  resume as libraries fully re-open over the summer. We will retain digital access to public domain materials
  in HathiTrust.
```
Also, the opening and closing single quotes aren't needed for string values.

Reply:
I wondered since we don't in some other places, but since the rest were in single quotes, I erred on the side of not messing w/the pattern.

Reply:
There are only a handful of cases where you need to use the quotes, and if you're checking the output, you'd notice it right away. But, yeah, consistency is nice.